### ISSUE TITLE
Please create a label for the MSI Pro Z960-A

### ISSUE THREAD

**Issue by handmeatowel** [Dasharo/dasharo-issues#101]
Please create a label for the MSI Pro Z960-A

**Comment by pietrushnic** [comment]
@handmeatowel that's the one, does it close the issue?

**Comment by handmeatowel** [comment]
Oh okay, that seems to be some MSI-internal version number. For someone who doesn't know that it'll probably counterintuitive. I'd assume most would look for _MSI Pro Z960-A_ :-)

**Comment by pietrushnic** [comment]
@handmeatowel that's correct, beause of that labels are typically assigned by 3mdeb staff.

**Comment by handmeatowel** [comment]
Just an suggestion for improvement then. Up to you, I'm aware of that now.

**Comment by miczyg1** [comment]
ms 7d25 would address a family of the boards:
- MSI PRO Z690-A DDR4 WIFI and non-WIFI
- MSI PRO Z690-A WIFI and non-WIFI (the DDR5 variant)

As they are pretty much the same (DDR5 VS DDR4 and WIFI VS NON-WIFI) I see no point in having 4 distinct labels

**Comment by handmeatowel** [comment]
Labelling it "MSI PRO Z690-A" would solve that issue :-)

**Comment by miczyg1** [comment]
> Labelling it "MSI PRO Z690-A" would solve that issue :-)

Actually no, because MSI PRO Z690-A is the explicit name of the DDR5 variant

**Comment by handmeatowel** [comment]
Oh okay!

**Comment by macpijan** [comment]
> Actually no, because MSI PRO Z690-A is the explicit name of the DDR5 variant

This is correct, if we want to be very strict. But maybe the label `MSI PRO Z690-A` could simply replace the `msi_ms7d25` and it would refer to all variants as well: WiFi / no WiFi / DDR4 / DDR5.

Above we have a suggestion that such label would make sense for end users to let them find what they need.

> Also maybe README should contain explanation of labels?

That is also an option, but having labels which are intuitively clear for end users should be a priority. We can still have in README, that e.g.  `MSI PRO Z690-A` label covers all variants, despite the fact that it is "the proper" name for DDR5 variant, as @miczyg1 pointed out above.

**Comment by zirblazer** [comment]
You may consider how future proof the tags system will be. The future MSI PRO Z790-P series is almost identical to the Z690-A but it uses a different internal name, 7E06 instead of 7D25. This name applies for both Z790-P DDR4 and DDR5 versions, even though they're actually slighty different (And the same applies for the Z690-A).

**Comment by miczyg1** [comment]
Changed the label to `MSI PRO Z690-A boards` hope it will be good. Similarly we will give a `MSI PRO Z790-P boards ` label to the next boards. Closing the issue
